### DISCUSSION TITLE
模型可以识别文字方向吗？

### DISCUSSION THREAD

**Discussion by touchsky-real** [PaddlePaddle/PaddleOCR#13325] [Q&A] [answered]
pdf文件里有一些页面方向不对，可以用paddleOCR转正这些页面吗？

**Comment by UserWangZz** [comment]
可以尝试使用predict_sys.py，添加cls分类，具体查看下面文档
https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/quickstart.md#211-%E4%B8%AD%E8%8B%B1%E6%96%87%E6%A8%A1%E5%9E%8B

**Comment by SWHL** [comment, marked as answer]
可以使用这个：https://github.com/RapidAI/RapidStructure/blob/main/docs/README_Orientation.md

**Reply by touchsky-real** [reply]
感谢！
